### DISCUSSION TITLE
✨ Kometa

### DISCUSSION THREAD

**Discussion by Snipees** [runtipi/runtipi-appstore#3762] [Requests]
### Description

Kometa is a powerful tool designed to give you complete control over your media libraries. With Kometa, you can take your customization to the next level, with granular control over metadata, collections, overlays, and much more.

### Official Website

https://github.com/Kometa-Team/Kometa

### Documentation link

https://kometa.wiki/en/latest/

### Last application release & date

2.0.2 (31/05/2024)

### Application license

MIT License

### Source code repository link

https://github.com/Kometa-Team/Kometa

### Docker image link

https://hub.docker.com/r/kometateam/kometa

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
@Snipees Great news, app is now available in store ;)

**Reply by JigSawFr** [reply]
Also don't hesitate to take a look here https://github.com/jmxd/Kometa or on their Discord, so many great examples !

**Reply by Snipees** [reply]
@JigSawFr Wonderful, thanks a lot !
I've been using it for a while now, originally called Plex Meta Manager.
Many many thanks !!

**Reply by JigSawFr** [reply]
I've configured it yesterday, working very great with the config linked before, so nice. Was aware of PMM before, but not enough time to try it 😁

**Reply by Snipees** [reply]
I've always used my own configuration, but this is an opportunity to test new ones... I'll take a look, thanks for the suggestion.
